src/main/java/io/wisetime/connector/ServerRunner.java

```diff
@@ -182,16 +217,34 @@ private ConnectorStore createStore(boolean persistenceRequired) {
       return new FileStore(persistentStoreDir);
     }
 
+    /**
+     * Implementation of {@link WiseTimeConnector} is required to start server.
+     */
     public ServerBuilder withWiseTimeConnector(WiseTimeConnector wiseTimeConnector) {
       this.wiseTimeConnector = wiseTimeConnector;
       return this;
     }
 
+    /**
+     * More information about WiseTime API key: <a href="https://wisetime.io/docs/connect/api/">WiseTime Connect API</a>.
+     * You have to provide api key or custom implementation of {@link ApiClient} for successful authorization.
+     * @see #withApiClient(ApiClient)
+     */
     public ServerBuilder withApiKey(String apiKey) {
       this.apiKey = apiKey;
       return this;
     }
 
+    /**
+     * Whether persistent storage is required for this connector. If true, this property forces the operator to set the
+     * DATA_DIR configuration parameter (via system property or environment variable). The DATA_DIR directory will be used
+     * to persist data across connector restarts.
+     *
+     * If false, the connector will not force the operator to specify a DATA_DIR. It will still use the DATA_DIR path if
+     * one is set. If none is provided, a subdirectory in the /tmp/ path will be used.
+     *
+     * Default is false.
```

Comment:
@sukolenvo I've taken the liberty to update the docs above. Can you please check that my understanding is correct?